docs/msbuild/removeduplicates-task.md

```diff
@@ -33,6 +33,7 @@ Removes duplicate items from the specified item collection.
 |Parameter|Description|
 |---------------|-----------------|
 |`Filtered`|Optional <xref:Microsoft.Build.Framework.ITaskItem>`[]` output parameter.<br /><br /> Contains an item collection with all duplicate items removed. The order of the input items is preserved, keeping the first instance of each duplicate item.|
+|`HadAnyDuplicates`|Optional `Boolean` output parameter.<br /><br /> If `true` any duplicate items were found, `false` otherwise.|
```

Comment:
Minor copy edit suggestion:

```suggestion
|`HadAnyDuplicates`|Optional `Boolean` output parameter.<br /><br /> If `true` duplicate items were found, `false` otherwise.|
```